ProcessMaker/Http/Controllers/TaskController.php

```diff
@@ -11,12 +11,16 @@ public function index()
   {
       return view('tasks.index');
   }
-//     public function edit(Tasks $tasks) /*<-----*/ // 
-//   {
-//     return view('tasks.edit',compact($tasks));  //is the edit page added to routes?
-//   }
-//   public function create()
-//   {
-//     return view('tasks.create');
-//   }
+    public function edit($id) 
```

Comment:
Remove edit. There is list and view only.

Reply:
removed